src/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixer.php

```diff
@@ -376,16 +376,26 @@ private function parseTypeHint(Tokens $tokens, int $index): array
     {
         $allowsNull = false;
 
-        if ($tokens[$index]->isGivenKind(CT::T_NULLABLE_TYPE)) {
-            $allowsNull = true;
-            $index = $tokens->getNextMeaningfulToken($index);
-        }
-
         $types = [];
 
         while (true) {
             $type = '';
 
+            if (\defined('T_READONLY') && $tokens[$index]->isGivenKind(T_READONLY)) { // @TODO: drop condition when PHP 8.1+ is required
```

Comment:
```suggestion
            if (\defined('T_READONLY') && $tokens[$index]->isGivenKind(T_READONLY)) { // @TODO: simplify condition when PHP 8.1+ is required
```

I guess only `defined` will get dropped